fbfmaproom/assets/style.css

```diff
@@ -42,11 +42,18 @@ tr {
     background-color: #ff6961;
 }
 
-.tooltiptext {
-  color: black;
-  position: relative;
-  background-color: rgba(228,228,228, 1);
-  max-width: 300px;
-  min-width: 300px;
-  font-size: 110%;
+.tooltiptext { 
+    opacity: 1 !important;
+}
+
+.tooltiptext .tooltip-inner {
+    color: black;
+    background-color: #E4E4E4;
+    max-width: 300px;
+    min-width: 300px;
+    font-size: 110%;
+}
```

Comment:
How about we use the same style for both `tooltip-inner` and `dash-table-tooltip`?

Reply:
I can try. It won't really be perfectly consistent because like I said they don't use the same structure. For example, the dash_table tooltips have a border and an entirely different shape.

Reply:
I see your comment in the edited PR description about not knowing if the DataTable CSS classes are intended to be part of the stable API. It's a good point, but I still think we should try to keep the two consistent. When we get around to writing front end tests, maybe we could write a test that verifies that the table tooltips have the expected classes?

Reply:
Don't spend all day on it. I'm just curious what happens if you change the selector on this style from `.tooltip-inner` to `.tooltip-inner, .dash-table-tooltip`. Then we'll move on, whether that works or not.

Reply:
Those are all good ideas. I'll try it in just a bit, I'm just going to push the table demo because the only thing it's "missing" is the updated environment and it'll be easy enough to rebase in terms of that

Reply:
Sorry, forgot to add this friday: So, the CSS class is `dash-tooltip` not `dash-table-tooltip` but in any case it doesn't work. Should we just call this done then? I tried a couple of other things too Friday afternoon and this morning and neither worked. At the very least in terms of consistency the table is no worse than it was before.

Reply:
Yes, let's move on. Out of curiosity, do you know why what you tried didn't work?

FWIW, I inspected the DOM in the browser and saw an element with the class `dash-table-tooltip`.

Reply:
I'm wondering what you were looking at then, I'm not entirely sure. I had to use the debugger to "freeze" the DOM to even inspect the tooltip. I did a lot of research on how to style these tooltips and the answers are conflicting and none worked. I think the bootstrap tooltip CSS just don't really work with the dash table ones because the structure is very different.